platform/io/platform-io-impl/src/main/java/org/codice/ddf/platform/io/FelixCfgPersistenceStrategy.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.platform.io;
+
+import static org.apache.commons.lang.Validate.notNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.util.Dictionary;
+import java.util.Enumeration;
+import java.util.Hashtable;
+
+import org.apache.felix.utils.properties.Properties;
+import org.codice.ddf.platform.io.internal.PersistenceStrategy;
+
+/**
+ * Class that persists configuration properties using the Felix cfg file format.
+ * <p>
+ * All values are stored as strings. The returned {@link Dictionary} from {@link #read(InputStream)}
+ * will always consist of string values that must be separately parsed or processed by the caller,
+ * even if they weren't strings when handed to {@link #write(OutputStream, Dictionary)}.
+ * <p>
+ * Not recommended for use with complex objects such as arrays or maps.
+ */
+public class FelixCfgPersistenceStrategy implements PersistenceStrategy {
+    @Override
+    public String getExtension() {
+        return "cfg";
+    }
+
+    @Override
+    public void write(OutputStream out, Dictionary<String, Object> properties) throws IOException {
+        notNull(out, "OutputStream cannot be null");
+        notNull(properties, "Properties cannot be null");
+        final Properties props = new Properties();
```

Comment:
✏️ I know I've been guilty of this exact same thing, but you might want to rename your parameter and/or your local variable. It's a bit of an eye-crosser to see:

- Dictionary properties
- Properties props
- properties.keys()
- props.put(key, properties.get(key))

A little naming clarity/differentiation would help a lot here.